Assets/_Application/Layers/UI/AddLayer/AddLayerPanel.cs

```diff
@@ -72,7 +72,7 @@ private IEnumerator AnimatePanel(bool open)
                 elapsedTime += Time.deltaTime;
                 if (open)
                 {
-                    var newY = Mathf.SmoothStep(rectTransform.sizeDelta.y, targetHeight, elapsedTime/smoothTime);
+                    var newY = Mathf.SmoothStep(rectTransform.sizeDelta.y, targetHeight, (elapsedTime/smoothTime)*2f);
```

Comment:
het is denk ik beter om de smoothTime in de inspector van de prefab aan te passen naar 0.5, dan blijft die zijn betekenis houden